hooks/boot/01-helpers.rb

```diff
@@ -10,7 +10,7 @@ def module_enabled?(kafo, name)
       mod.enabled?
     end
 
-    def log_and_say(level, message)
+    def log_and_say(level, message, log_only = false)
```

Comment:
I thought about that a bit, it seemed logical but that it adds furthering branching if/else elsewhere and I couldn't decide if the cost was worth it. So thats me saying I could go either way if @ekohl and @chris1984 have a preference.